WebView/app/src/main/java/com/android/samples/webviewdemo/MainActivity.kt

```diff
@@ -196,10 +189,8 @@ class MainActivity : AppCompatActivity() {
         title = getString(R.string.app_name)
 
         // Setup debugging; See https://developers.google.com/web/tools/chrome-devtools/remote-debugging/webviews for reference
-        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.KITKAT) {
```

Comment:
I assume this is because of minSdkVersion?

Reply:
Yeah, since the minSdkVersion is 21, it's always >= 19 :)